assets/templates/partials/calendar/filters.tmpl

```diff
@@ -0,0 +1,55 @@
+<div class="ons-accordion ons-pl-grid-col">
+  <div class="ons-u-fs-r--b">Filter results</div>
+
+  <div class="ons-collapsible ons-js-collapsible ons-collapsible--accordion" data-btn-close="Hide" data-group="accordion" data-open="true">
+    <div class="ons-collapsible__heading ons-js-collapsible-heading">
+      <div class="ons-collapsible__controls">
+        <h2 class="ons-collapsible__title">Release type</h2>
+        <span class="ons-collapsible__icon">
+          <svg class="ons-svg-icon" viewBox="0 0 8 13" xmlns="http://www.w3.org/2000/svg" focusable="false" fill="currentColor">
+            <path d="M5.74,14.28l-.57-.56a.5.5,0,0,1,0-.71h0l5-5-5-5a.5.5,0,0,1,0-.71h0l.57-.56a.5.5,0,0,1,.71,0h0l5.93,5.93a.5.5,0,0,1,0,.7L6.45,14.28a.5.5,0,0,1-.71,0Z" transform="translate(-5.02 -1.59)" />
+          </svg>
+        </span>
```

Comment:
Icons are best in a separate `assets/partials/icons` directory. This particular icon is in the `dp-renderer` https://github.com/ONSdigital/dp-renderer/blob/d5ba75b3597c7dc56d61b6e4f4c00a520c576119/assets/templates/icons/collapsible.tmpl